app/policies/invite_request_policy.rb

```diff
@@ -0,0 +1,11 @@
+class InviteRequestPolicy < ApplicationPolicy
+  MANAGE_ROLES = %w[superadmin policy_and_abuse support].freeze
+
+  def can_manage?
+    user_has_roles?(MANAGE_ROLES)
+  end
+
+  alias can_destroy? can_manage?
```

Comment:
Is `can_destroy?` called anywhere?

Reply:
I have utterly no idea why I put that there. I'll remove it.